test/unit/index.spec.js

```diff
@@ -0,0 +1,15 @@
+const rewire = require('rewire');
+
+const server = require('../../src/server');
+const containers = require('../../src/containers');
+
+describe('index', () => {
+  it('start server and containers', () => {
+    sinon.stub(server, 'listen');
+    sinon.stub(containers, 'startUp');
+
+    rewire('../../src/index');
+    expect(server.listen.callCount).to.equal(1);
+    //expect(containers.startUp.callCount).to.equal(1);
```

Comment:
In or out?

Reply:
I don't know. While talking about it, we decided to keep it in, right? 
This is part of the discussion of how we deliver this in the first iteration and what seem like reasonable instructions for users.

Reply:
Yeah I think keep it (ie: uncomment the assertion).

Reply:
Ok, I added a bunch of code around that.